include/deal.II/grid/tria_accessor.templates.h

```diff
@@ -2233,9 +2238,14 @@ TriaAccessor<0, dim, spacedim>::operator ++ () const
 template <int dim, int spacedim>
 inline
 void
-TriaAccessor<0, dim, spacedim>::operator -- () const
+TriaAccessor<0, dim, spacedim>::operator -- ()
 {
-  Assert (false, ExcNotImplemented());
+  if (global_vertex_index!=numbers::invalid_unsigned_int)
+    {
+      --global_vertex_index;
+      if (global_vertex_index==static_cast<unsigned int>(-1))
+        global_vertex_index = numbers::invalid_unsigned_int;
```

Comment:
I think I should have been clearer. This, too, only works accidentally because `global_vertex_index` happens to have data type `unsigned int`. My idea was to write it like this:
```
  if (global_vertex_index > 0)
    --global_vertex_index;
  else
    global_vertex_index = numbers::invalid_unsigned_int;
```